src/Symfony/Component/Process/Process.php

```diff
@@ -1198,6 +1214,30 @@ public function setEnhanceSigchildCompatibility($enhance)
     }
 
     /**
+     * Sets whether environment variables will be inherited or not.
+     *
+     * @param bool $inheritEnv
+     *
+     * @return ProcessBuilder
+     */
+    public function inheritEnvironmentVariables($inheritEnv = true)
+    {
+        $this->inheritEnv = (bool) $inheritEnv;
+
+        return $this;
+    }
+
+    /**
+     * Returns whether environment variables will be inherited or not.
+     *
+     * @return bool
+     */
+    public function getInheritEnvironmentVariables()
```

Comment:
this sounds like a getter for Inherited env vars. What about `areEnvironmentVariablesInherited`?